Docs/source/libensemble/libensemble.rst

```diff
@@ -1,3 +1,5 @@
+.. _libensemble:
```

Comment:
Thanks for this PR @ax3l !
I was wondering, would it also make sense to add a link to the SUMMIT installation described in `Docs/source/building/summit.rst` to this file in the section of installation?

Reply:
This section should be general for runs on laptop and Summit, but we can certainly back-link :)

Reply:
updated accordingly.